IRCCloud logo pushed outside of top of page.

Anzrael opened the pull request:
Having the header margin at 0 was pushing the logo outside of the top of the screen. Changing it to 10px aligns the logo to sit vertically in the middle of the header section.
![screenshot_022515_042855_pm](https://cloud.githubusercontent.com/assets/6538746/6383068/659e2942-bd0b-11e4-92cc-1b0417b6f745.jpg)

![screenshot_022515_042936_pm](https://cloud.githubusercontent.com/assets/6538746/6383096/7da765b2-bd0b-11e4-8632-1a1f457b3f63.jpg)

Anzrael committed:
Changed header title margin

Having the header margin at 0 was pushing the logo outside of the top of the screen. Changing it to 10px aligns the logo to sit vertically in the middle of the header section.

zvuc commented:
I do remember adding a line of `margin-top: 0` for `h1#title` a few days ago (here : 2cb2b0de2323437b8eed71c0509028223012907a) because it suddenly was pushed down 10 pixels. I've tested on both FF and Chrome and is looking fine for me with the latest code I've pushed to userstyles. Looking at the screenshots it seems like you've modified the default fonts, could there by any other place you might have changed?

Anzrael commented:
I have changed the default font locally on my machine since I do prefer monospaced fonts for IRC. I reverted my installation of Aquamarine to be what is on userstyles.org and this is the end result:

![screenshot_022615_114119_am](https://cloud.githubusercontent.com/assets/6538746/6398937/75edace0-bdac-11e4-8a38-f32941b6a268.jpg)

The logo is still being pushed outside of the top of the screen. Inspecting the CSS on the page shows that IRCCloud applies a margin-top of 10px that is overwritten by Aquamarine to be 0. Reverting it back to the 10px aligns the logo properly. Note that this is in Chrome and I haven't checked how it renders in Firefox but I imagine that it would come to the same result.

zvuc commented:
I could not reproduce this problem on my end as reverting the 0px override to 10px pushes down the logo. Instead I just rewrote all the code with static height values and all to reposition everything in the latest commit 9b5f53e6b3522be5f16e7c6730d6b5540fbf27f4. Try updating and let me know how it goes.
